insights/specs/insights_archive.py

```diff
@@ -128,6 +128,7 @@ class InsightsArchiveSpecs(Specs):
     lsblk_pairs = simple_file("insights_commands/lsblk_-P_-o_NAME_KNAME_MAJ_MIN_FSTYPE_MOUNTPOINT_LABEL_UUID_RA_RO_RM_MODEL_SIZE_STATE_OWNER_GROUP_MODE_ALIGNMENT_MIN-IO_OPT-IO_PHY-SEC_LOG-SEC_ROTA_SCHED_RQ-SIZE_TYPE_DISC-ALN_DISC-GRAN_DISC-MAX_DISC-ZERO")
     lscpu = simple_file("insights_commands/lscpu")
     ls_lib_firmware = simple_file("insights_commands/ls_-lanR_.lib.firmware")
+    lsinitrd = simple_file("insights_commands/lsinitrd")
```

Comment:
Since this file is also collected in the sosreport, better add it to sos_archive.py together.

Reply:
Added.